packages/components/pages/f-contact-preferences/src/components/ContactPreferences.vue

```diff
@@ -142,8 +147,25 @@ export default {
         ])
     },
 
+    watch: {
+        immediate: true,
```

Comment:
Does this work correctly? I thought `immediate` had to be inside the bit you're watching, and not outside.

Reply:
There is an alternative way ... I will try that 👍